tests/framework/interface.go

```diff
@@ -39,6 +39,7 @@ type Member interface {
 	Stop()
 }
 
+// Client is a wrapper on top of "go.etcd.io/etcd/client/v3".Client interface
```

Comment:
Not sure what what `wrapper on top of ... interface` means. I would set that this interface is meant as a subset of `clientv3.Client` interface implemented by common framework, however this is also not true as options are passed differently to be more expressive. 

Overall this is still work in progress, so I would propose that you skip adding documentation in this PR.